sup3r/utilities/regridder.py

```diff
@@ -499,6 +496,10 @@ def __init__(self, source_files, out_pattern, target_meta, heights,
                                        cache_pattern=cache_pattern,
                                        n_chunks=n_chunks,
                                        max_workers=self.query_workers)
+        DistributedProcess.__init__(self, max_nodes=max_nodes,
```

Comment:
Oh wow i didnt know you could just call the inherited init directly and pass in self! that makes so much sense. Nice.

Reply:
Thanks! Didn't know this worked either just figured it _should_ lol